fsharp-backend/src/LibService/Rollbar.fs

```diff
@@ -450,5 +450,7 @@ let init (serviceName : string) : unit =
   //   )
   // |> ignore<obj>
 
+  Prelude.rollbarExceptionInBackground <- fun msg metadata -> sendError msg metadata
```

Comment:
I don't follow this - are these calls safe?

The name implies async but it's not, right? It's just a regular rollbar call?

Reply:
We use "background" at other time to mean async, right? I'd prefer a different name. Certainly needs a comment.

The other thing I was unclear about was whether this could/would throw an exception.

Perhaps: `Prelude.sendRollbarError`?